src/main/java/reposense/report/RepoCloner.java

```diff
@@ -66,7 +68,15 @@ public RepoLocation getClonedRepoLocation(String outputPath) throws IOException
             return null;
         }
 
-        currentRepoDefaultBranch = GitBranch.getCurrentBranch(configs[currentIndex].getRepoRoot());
+        try {
+            currentRepoDefaultBranch = GitBranch.getCurrentBranch(configs[currentIndex].getRepoRoot());
```

Comment:
Actually on a second look, this is not a very good solution.
Your proposed message mentioned that GitBranch is the cause of the crash, but the solution is applied on the RepoCloner. This means that any subsequent use of the `GitBranch.getCurrentBranch` elsewhere would still result in a crash, making it an unsafe api.

Can you tweak the solution? Probably make the `GitBranch.getCurrentBranch` to throw a checked exception instead to enforce the use of this API handle the exception.